webcompat/static/css/webcompat.dev.css

```diff
@@ -40,18 +40,18 @@
 
 #body-webcompat {
   background-color: var(--color-bg-page);
-  margin-top: var(--navi-height);
+  padding: var(--navi-height) var(--grid-edge-s) 25px;
 }
 
 @media screen and (min-width: 665px) {
   #body-webcompat {
-    padding: var(--navi-height) 19px 50px;
+    padding: var(--navi-height) var(--grid-edge-m) 50px;
```

Comment:
I know, we used a fix value here, but could we change the `50px` as we are on it anyway to `calc(--unit-gutter-m * 2)`?